main.go

```diff
@@ -48,13 +48,15 @@ type BaseConfig struct {
 
 type RDSConfig struct {
 	BaseConfig `yaml:"base,inline"`
-	Regions    []string `yaml:"regions"`
+	Regions    []string      `yaml:"regions"`
+	Interval   time.Duration `yaml:"interval"`
```

Comment:
This is probably something all collectors have, so you can add the `Interval` into the `BaseConfig` only once and it is included in every `Config` that will include it (which all configs should).

Reply:
Added it to every config, cause some might have a lower interval, cause they need less time for collection.

Reply:
That should still work, as the baseconfig is just merged in - so every config has the field and can set it to a different value.

Reply:
Missed that, thanks for pointing it out! I"ll update the PR.